src/lib.rs

```diff
@@ -8,6 +8,11 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#[macro_use]
```

Comment:
Ooh, I see "using slog in library does not force users of the library to use slog (but gives benefits); see crates/example-lib" on the `slog` documentation page.  That's awesome sauce!